### PR TITLE
Add step "cucushift-upgrade-setedge-2hops"

### PR DESCRIPTION
**Pull request by liangxia**
Follow up of https://github.com/openshift/release/pull/58349, but for 2 hops.

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-cpou-upgrade-from-4.10.yaml [hunk 1]
```diff
@@ -55,12 +49,6 @@ releases:
       channel: candidate
       version: "4.10"
   target:
-    candidate:
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-cpou-upgrade-from-4.10.yaml:58]: There are several steps is injecting `cli: target`, and it need amd-arch cli to run in build farm pod. So I guess the `target` image can not be changed to arm payload.

```
$ grep -r "cli: target"
ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/upgrade/rollback/cucushift-upgrade-rollback-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/upgrade/rhel-worker/cucushift-upgrade-rhel-worker-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/upgrade/setchannel/cucushift-upgrade-setchannel-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/upgrade/toversion/cucushift-upgrade-toversion-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/upgrade/check-not-recommended/cucushift-upgrade-check-not-recommended-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-ref.yaml:  cli: target
ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-ref.yaml:  cli: target
ci-operator/step-registry/openshift-extended/upgrade/post/openshift-extended-upgrade-post-ref.yaml:  cli: target
```

The issues also happened in https://github.com/openshift/release/pull/58349 (Sorry to be late)

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-cpou-upgrade-from-4.10.yaml]: I am thinking if we want to re-use this step in both arm and amd ci, maybe we can introduce `release:final` image, and always keep `release:target` in ci yaml file.

In arm ci yaml file, we need to rename `release:target-arm64` to `release:final`
in amd ci yaml file, we need to introduce `release:final` image which pointed to amd release image.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-cpou-upgrade-from-4.10.yaml]: I'll use another PR to track this change, to make PR(s) small and simple

**Comment by jiajliu** [comment]
lgtm

**Pull request merged**